src/main.c

```diff
@@ -2582,8 +2583,61 @@ void reset_model() {
     glfwSetTime(g->day_length / 3.0);
     g->time_changed = 1;
 }
+void display(void)
```

Comment:
Functions in main could benefit from doxygen comments explaining what each one does.